CMakeLists.txt

```diff
@@ -109,7 +108,9 @@ target_include_directories(fv3dycore INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT
                                                $<INSTALL_INTERFACE:mod>)
 
 target_link_libraries(fv3dycore PUBLIC fms
-                                       gfsphysics
```

Comment:
I am curious, we only had noahmp_tables.f90 in gfsphysics directory, does fv3dycore have dependency on that file? Can we remove this line?

Reply:
The dycore has dependencies on CCPP because of the fast physics calls (saturation adjustment) in `fv_mapz.F90`. It won't compile w/o these dependencies. The `gfsphysics` line had the CCPP dependencies included implicitly because of line 126 below in the old code.

Reply:
So fv3dycore has no dependency on gfsphysics itself, previous code lists gfsphysics not ccpp as dependency, maybe ccpp physics can be listed here explicitly? Also do we need ccpp? It seems the dependency is on ccppdata and ccppphys.

Reply:
The new code has dependencies ccppdata (for the data structures), ccppphys for the fast physics code, and ccpp (=ccpp-framework) for the CCPP API:
```
target_link_libraries(fv3dycore PUBLIC fms
                                       ccpp
                                       ccppphys
                                       ccppdata
                                       esmf)
```

Reply:
Sorry I don't get the dependency of ccpp (ccpp-framwork). I run the GFSv16 test without the line of ccpp, the code runs fine. 
 target_link_libraries(fv3dycore PUBLIC fms
-                                       ccpp
                                        ccppphys
                                        ccppdata
                                        esmf)

Reply:
I can try. Maybe it's yet another implicit dependency in ccppphys.

Reply:
It works without that line, too. But I am wondering if it is the right thing to do instead of spelling out all dependencies explicitly. @DusanJovic-NOAA what do you think?

Reply:
If `fv3dycore` does not depend directly on `ccpp` (ie. ccpp-framwork), then I would leave it out. BTW at some point we should rename it from `ccpp` to `ccppframework` to be more explicit about what it is.

Reply:
The dycore does depend explicitly on ccpp=ccppframework, because it uses `call ccpp_physics_init` etc.

Reply:
In that case, specify `ccpp` as explicit dependency.

Reply:
Ok, thanks.